main.go

```diff
@@ -53,20 +58,28 @@ func main() {
 		glog.Fatalf("Error building kubernetes clientset: %s", err.Error())
 	}
 
-	exampleClient, err := clientset.NewForConfig(cfg)
+	controllerClient, err := clientset.NewForConfig(cfg)
 	if err != nil {
 		glog.Fatalf("Error building example clientset: %s", err.Error())
 	}
 
-	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(kubeClient, time.Second*30)
-	exampleInformerFactory := informers.NewSharedInformerFactory(exampleClient, time.Second*30)
+	kubeInformersNamespaced := kubeinformers.NewFilteredSharedInformerFactory(kubeClient, 10*time.Minute, utility.GetControllerNamespace(), nil)
+	controllerInformerFactory := informers.NewSharedInformerFactory(controllerClient, 10*time.Minute)
```

Comment:
As `EtcdStorage` type is Cluster-scoped, do we need `NewFilteredSharedInformerFactory` approach here?

Reply:
> As EtcdStorage type is Cluster-scoped, do we need NewFilteredSharedInformerFactory approach here?

You do not.